InternxtDesktop/Views/Backups/BackupContentNavigator.swift

```diff
@@ -150,17 +151,75 @@ struct BackupContentNavigator: View {
     }
     
     private func downloadItem() {
-        let alert = NSAlert()
-        alert.messageText = "Download not implemented yet"
-        alert.informativeText = "You triggered a download, but this feature is not yet implemented"
         
-        alert.runModal()
+        
+        if(backupsService.backupDownloadStatus == .InProgress) {
+            let alert = NSAlert()
```

Comment:
move this to a file called `BackupUtils` or similar, I believe we are going to reuse this somewhere else